maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationService.java

```diff
@@ -40,7 +41,9 @@ public ApplicationDTO create(WorkflowRequest request) {
 
         CourtType courtType = request.getCourtType();
         Action action = (courtType == CourtType.MAGISTRATE) ? Action.CREATE_MAGS_HARDSHIP : Action.CREATE_CROWN_HARDSHIP;
-        validate(request, action);
+        int repId = request.getApplicationDTO().getRepId().intValue();
+        RepOrderDTO repOrderDTO = maatCourtDataService.findRepOrder(repId);
```

Comment:
We can extract line 44-46 to a common method, say getRepOrderDTOAndValidate() as they are used in the 'create' and 'update'

Reply:
Fixed